[2024-04-29T16:27:04.532+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-29T16:27:04.783+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fill_data_v5.test scheduled__2024-04-28T00:00:00+00:00 [queued]>
[2024-04-29T16:27:04.800+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fill_data_v5.test scheduled__2024-04-28T00:00:00+00:00 [queued]>
[2024-04-29T16:27:04.801+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-29T16:27:04.830+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): test> on 2024-04-28 00:00:00+00:00
[2024-04-29T16:27:04.841+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fill_data_v5', 'test', 'scheduled__2024-04-28T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/dag_raw_processed.py', '--cfg-path', '/tmp/tmpr0_dq162']
[2024-04-29T16:27:04.845+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask test
[2024-04-29T16:27:04.845+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=562) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-29T16:27:04.847+0000] {standard_task_runner.py:63} INFO - Started process 564 to run task
[2024-04-29T16:27:04.904+0000] {task_command.py:426} INFO - Running <TaskInstance: fill_data_v5.test scheduled__2024-04-28T00:00:00+00:00 [running]> on host 8865bf60ce19
[2024-04-29T16:27:04.994+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mosula' AIRFLOW_CTX_DAG_ID='fill_data_v5' AIRFLOW_CTX_TASK_ID='test' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-28T00:00:00+00:00'
[2024-04-29T16:27:04.996+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-29T16:27:05.012+0000] {python.py:237} INFO - Done. Returned value was: []
[2024-04-29T16:27:05.013+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-29T16:27:05.038+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fill_data_v5, task_id=test, execution_date=20240428T000000, start_date=20240429T162704, end_date=20240429T162705
[2024-04-29T16:27:05.064+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-29T16:27:05.108+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-29T16:27:05.111+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-04-29T16:39:38.643+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-29T16:39:38.766+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fill_data_v5.test scheduled__2024-04-28T00:00:00+00:00 [queued]>
[2024-04-29T16:39:38.773+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fill_data_v5.test scheduled__2024-04-28T00:00:00+00:00 [queued]>
[2024-04-29T16:39:38.774+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-29T16:39:38.784+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): test> on 2024-04-28 00:00:00+00:00
[2024-04-29T16:39:38.791+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fill_data_v5', 'test', 'scheduled__2024-04-28T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_raw_processed.py', '--cfg-path', '/tmp/tmpd38ukp44']
[2024-04-29T16:39:38.793+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=202) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-29T16:39:38.793+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask test
[2024-04-29T16:39:38.794+0000] {standard_task_runner.py:63} INFO - Started process 204 to run task
[2024-04-29T16:39:38.833+0000] {task_command.py:426} INFO - Running <TaskInstance: fill_data_v5.test scheduled__2024-04-28T00:00:00+00:00 [running]> on host 15fcc90b2790
[2024-04-29T16:39:38.896+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mosula' AIRFLOW_CTX_DAG_ID='fill_data_v5' AIRFLOW_CTX_TASK_ID='test' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-28T00:00:00+00:00'
[2024-04-29T16:39:38.897+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-29T16:39:38.920+0000] {python.py:237} INFO - Done. Returned value was: ['data.csv']
[2024-04-29T16:39:38.921+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-29T16:39:38.936+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fill_data_v5, task_id=test, execution_date=20240428T000000, start_date=20240429T163938, end_date=20240429T163938
[2024-04-29T16:39:38.970+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-29T16:39:38.992+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-29T16:39:38.995+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
